tests/helper.js

```diff
@@ -1,6 +1,10 @@
-const { DeviceLists, RequestType, KeysUploadRequest, KeysQueryRequest } = require("../pkg");
+import { expect, beforeAll } from "vitest";
+import { DeviceLists, RequestType, KeysUploadRequest, KeysQueryRequest } from "../";
+import { initAsync } from "../";
 
-function* zip(...arrays) {
+beforeAll(() => initAsync());
```

Comment:
Why this here?

Reply:
Oh, it's because `tests/helper.js` is defined as a setup file in vitest config. Maybe a comment would be welcomed.

Reply:
This was removed